dracula-theme.el

```diff
@@ -56,7 +55,8 @@
    'dracula
    ;; default
    `(cursor ((,class (:background ,fg3))))
-   `(default ((,class (:background ,bg1 :foreground ,fg1))))
+   `(default ((((type nil)) (:background "#000000" :foreground ,fg1))
```

Comment:
That answers my question, thanks! 🙂

I was just wondering how the code was evaluated. I was guessing it would short circuit, but wasn't sure. Thanks!